runtime/polkadot/src/lib.rs

```diff
@@ -1231,6 +1395,60 @@ impl OnRuntimeUpgrade for TechnicalMembershipStoragePrefixMigration {
 	}
 }
 
+/// Set the initial host configuration for Polkadot.
+pub struct SetInitialHostConfiguration;
+impl OnRuntimeUpgrade for SetInitialHostConfiguration {
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		use parachains_configuration::HostConfiguration;
+
+		let active_config: HostConfiguration<BlockNumber> = HostConfiguration {
+			max_code_size: 10_485_760,
+			max_head_data_size: 20_480,
+			max_upward_queue_count: 10,
+			max_upward_queue_size: 51_200,
+			max_upward_message_size: 51_200,
+			max_upward_message_num_per_candidate: 10,
+			hrmp_max_message_num_per_candidate: 10,
+			validation_upgrade_frequency: 14_400,
+			validation_upgrade_delay: 600,
+			max_pov_size: 5_242_880,
+			max_downward_message_size: 51_200,
+			ump_service_total_weight: 100_000_000_000,
+			hrmp_max_parachain_outbound_channels: 10,
+			hrmp_max_parathread_outbound_channels: 0,
+			_hrmp_open_request_ttl: 72,
+			hrmp_sender_deposit: deposit(1004, 100 * 1024),
+			hrmp_recipient_deposit: deposit(1004, 100 * 1024),
```

Comment:
This will be a massive deposit. 20,000 DOT at least (1004 * 20 * DOLLARS), then we also have to add cost of bytes.

How was this determined?

Reply:
Yeah, that's a TODO - it's the same as the Kusama values which are themselves too large.